monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -4191,8 +4191,11 @@ object Task extends TaskInstancesLevel1 {
     */
   final case class Options(
     autoCancelableRunLoops: Boolean,
-    localContextPropagation: Boolean
+    localContextPropagation: Boolean,
+    contextIsolationOnFork: Boolean
```

Comment:
If we go through with it, we could also add env variable to https://github.com/monix/monix/blob/master/monix-execution/jvm/src/main/scala/monix/execution/internal/Platform.scala

Reply:
Yep, if we go with this approach, we should do it for consistency